Add Firefox release automation script

pepelsbey opened the pull request:
### Description

It’s an automation script that helps to create Firefox release notes.

I created the [Firefox 145 release notes PR](https://github.com/mdn/content/pull/41896) using this script.

### Motivation

To simplify release notes workflow.

### Additional details

How it works, for example, with the newly released Firefox 145:

```sh
node scripts/content/release-firefox.js 145
```

What it does:

Takes the newly released version as a parameter (N), and:

- Removes active from N-1
- Updates N to Stable
  - Uncomments HTML, CSS, JavaScript “no notable changes” sections
  - Removes all other unused commented sections
- Updates N+1 to Beta
- Adds Nightly N+2 page

For release dates, it uses official JSON API — https://whattrainisitnow.com/api/firefox/calendar/future/

### Related issues and pull requests

- https://github.com/mdn/content/pull/41896

bsmth commented on helpers/release-firefox.js:
```diff
@@ -0,0 +1,384 @@
+ *
+ * Usage: node scripts/release-firefox.js VERSION
+ *
+ * Example: node scripts/release-firefox.js 145
```
Currently, this is wrong, but it's a good idea to have `./scripts/content` or `./scripts/author-utils` or so.

pepelsbey commented:
@bsmth Thanks! I implemented your suggestions:

- Moved script to the `scripts/content` subfolder.
- Changed CODEOWNERS.
- Split template into a Markdown file.

It looks ready for the upcoming release project :)

caugner commented on scripts/content/release-firefox.js:
```diff
@@ -0,0 +1,305 @@
+ * Fetch release dates from whattrainisitnow.com
+ */
+async function fetchReleaseDates() {
+  const futureReleases = await fetchJSON(CALENDAR_API);
```
Alternatively:

```suggestion
  const futureReleases = await fetch(CALENDAR_API).then(res => res.json());
```
